dcrpm/rpmutil.py

```diff
@@ -172,6 +189,33 @@ def check_rpmdb_indexes(self):
             "Triggername": None,  # rarely used
         }
 
+        # For some platforms, the command and/or checks need to be tweaked
```

Comment:
For future people, please comment on which each one of these is different (there's no conflict for X, but there is for Y, that conflict is P)